engine/src/main/java/org/terasology/rendering/dag/nodes/CopyImageToScreenNode.java

```diff
@@ -35,7 +35,7 @@
 import static org.terasology.rendering.world.WorldRenderer.RenderingStage.MONO;
 
 public class CopyImageToScreenNode extends ConditionDependentNode implements FBOManagerSubscriber {
-    private static final ResourceUrn DEFAULT_FRAME_BUFFER_URN = new ResourceUrn("engine:display");
+    private static final ResourceUrn DEFAULT_FBO = new ResourceUrn("engine:display");
```

Comment:
I specifically want to point this one out, because it might seem like the new name is actually less descriptive. However, all the Nodes (finally!) follow a common naming scheme, and all the ResourceUrns to Buffers are named as NAME_FBO.